.github/workflows/python-publish.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will upload a Python Package using Twine when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Upload Python Package
+
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: read
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: '3.x'
```

Comment:
I noticed there is no tox.ini in pysisu repo, suggesting adding it to ensure it is capatible with all major 3.x releases, at this moment should cover: 3.8/3.9/3.10, these 3 versions have 3+ years of active support.

Reply:
This line should be:
```
 python-version: ["3.8", "3.9", "3.10"]
```

Reply:
Instead of tox.ini I was thinking of using poetry instead. 

@leongenyi actually wrote up a doc on this https://www.notion.so/sisudata/Public-API-Protos-Packages-981cccc118554482aa8d854e1d2bbaf8#19445c4a5f884a71a92abc6b07d1e8e0

He doesnt include tox.ini, but does talk about how it seems like the community is moving to poetry.

This is also consistent with our infrastructure as well

Reply:
As far as the versions go, I agree, but why stop at 3.8? Why not also include 3.7?

Reply:
tool choice aside, how do we ensure that this works with these python versions/

Reply:
wayne i know at least 3.7 should be in there because that is the version we use too

Reply:
Yes, we should add 3.7 support for now

I also support using poetry to embed tox configuration, let's go for it!

Reply:
So it looks like the we need to keep it as it is, but we actually specify the deploy version in the setup.py. We can do this when we move to poetry.

see 
https://github.com/actions/setup-python

I updated the [ticket here](https://app.asana.com/0/1202238984240820/1202626170309678/f) to include this